slips_files/core/helpers/whitelist.py

```diff
@@ -552,6 +553,31 @@ def read_whitelist(self):
                                 # first time seeing this org
                                 whitelisted_orgs[data] = whitelist_line_info
 
+                        elif "whitelist" in type_:
```

Comment:
based on the example you put of the wildcard in the whitelist.conf. this condition is never triggered.